tools/ARIAtools/extractProduct.py

```diff
@@ -526,6 +585,25 @@ def merged_productbbox(
         proj = ds.GetProjection()
         ds = None
 
```

Comment:
If the user bbox is larger than the initial bbox, that means the user knows what (s)he is doing, and extract from scratch.

Reply:
We should warn the user if the new bbox is much smaller than the old one.
* Don't like prompting because it reduces automation.
* One option: If the new bbox is way smaller than the previous one, kick out the products smaller than the previous bbox
* If the user passes an explicit AOI, trust the user and use the new (smaller) bbox